chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/developer/attestation/DeveloperAttestationReportDataCollection.java

```diff
@@ -96,6 +72,7 @@ public DeveloperAttestationReportDataCollection(DeveloperDAO developerDAO,
             .collect(Collectors.toList());
 
     public List<DeveloperAttestationReport> collect(List<Long> selectedAcbIds, Logger logger) {
+        /*
```

Comment:
needed?

Reply:
This is code that is "broken" until we fix the reports in ticket OCD-3980.  Left it in to mark where we need to start, when fixing the reports.